### DISCUSSION TITLE
June 20th 2024 PowerShell/OpenSSH Community Call

### DISCUSSION THREAD

**Discussion by SydneyhSmith** [PowerShell/PowerShell#23892] [Community Call]
Agenda

- PSConfEU @dcaro 
- DSC/WinGet @mgreenegit 
- Build recap @SydneyhSmith 
- PowerShell 7 release @adityapatwardhan 
- Code of Conduct update @SydneyhSmith 
- Docs update @sdwheeler 

Please add any topics/demos/questions

**Comment by StevenBucher98** [comment]
docs update
